tdrs-backend/tdpservice/parsers/parse.py

```diff
@@ -55,22 +55,27 @@ def parse_datafile(datafile):
 
     return errors
 
-
 def bulk_create_records(unsaved_records, line_number, header_count, batch_size=10000, flush=False):
     """Bulk create passed in records."""
     if (line_number % batch_size == 0 and header_count > 0) or flush:
         logger.debug("Bulk creating records.")
         try:
-            num_created = 0
-            num_expected = 0
-            for model, records in unsaved_records.items():
-                num_expected += len(records)
-                num_created += len(model.objects.bulk_create(records))
-            if num_created != num_expected:
-                logger.error(f"Bulk create only created {num_created}/{num_expected}!")
+            num_records_created = 0
```

Comment:
Hi Eric. I've been rereading this over and over... The code looks right to me. All my comments are on clarity. First, as you may know, an ES "document" is synonymous with a record in this context. ("Documents are JSON objects that are stored within an Elasticsearch index and are considered the base unit of storage. In the world of relational databases, documents can be compared to a row in table.") So you could name all the variables as "records" (or "recs"). Also, it seems we're tracking creates in three resources. I think it would help to name the resources, so maybe something like: 

- num_expected_records --> postgres_records
- num_records_created --> django_records
- num_documents_created --> elastic_records